src/js/view/controls/next-display-icon.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'utils/helpers',
+    'utils/ui',
+    'templates/display-icon.html'
+], function(utils, UI, Template) {
+
+    return class NextDisplayIcon {
+
+        constructor(model, api) {
+            const element = utils.createElement(Template({
+                iconName: 'next',
+                ariaLabel: model.get('localization').next
+            }));
+
+            this.iconUI = new UI(element).on('click tap', function() {
```

Comment:
In `dock.js` you have a setup function which is called in the constructor, here you do everything in the constructor. Not sure which is better but should we choose 1 style and stick to it?